agent/hibinit-agent

```diff
@@ -41,6 +41,7 @@ HIB_ENABLED_FILE = "hibernation-enabled"
 IMDS_BASEURL = 'http://169.254.169.254'
 IMDS_API_TOKEN_PATH = 'latest/api/token'
 IMDS_SPOT_ACTION_PATH = 'latest/meta-data/hibernation/configured'
+MAX_SWAP_SIZE_OFFSET_ALLOWED = 100 * 1024 
```

Comment:
Why 100K is max offset?

Reply:
The offset is kept just to make sure that on every fresh start we are not recreating the swap. It is a limit that I am proposing to avoid unnecessary deletion of swap. But let me know if you feel otherwise